riva/clients/nmt/streaming_s2s_client.cc

```diff
@@ -301,21 +326,33 @@ StreamingS2SClient::ReceiveResponses(std::shared_ptr<S2SClientCall> call, bool a
       std::cout << "ASR started... press `Ctrl-C' to stop recording\n\n";
       gotoxy(0, 5);
     }
-    // Write to WAV file
+
     std::cout << "Got " << audio.length() << " bytes back from server" << std::endl;
-    // different for opus
-
-    int16_t* audio_data = (int16_t*)audio.data();
-    size_t len = audio.length() / sizeof(int16_t);
-    std::copy(audio_data, audio_data + len, std::back_inserter(pcm_buffer));
-    if (audio.length() < TTS_CHUNK_SIZE_BYTES) { // last chunk for this request
-      ::riva::utils::wav::Write(
-          std::string("streaming_" + std::to_string(cur_response) + ".wav"), 44100,
-          pcm_buffer.data(), pcm_buffer.size());
-      cur_response++;
-      pcm_buffer.clear();
+    if (tts_encoding_.empty() || tts_encoding_ == "pcm") {
```

Comment:
can get rid of tts_encoding_.empty() check (here and few other places) as we check it at the beginning.

Reply:
empty is allowed, if the user doesn't pass the param. Actually, in the beginning, we are checking for error, i.e. if the encoding is not empty and anything other than pcm/opus are provided.